### PR TITLE
feat: add footer partners

### PR DESCRIPTION
**Pull request by gary-van-woerkens**

### CODE REVIEW

File: src/Footer.tsx [hunk 1]
```diff
@@ -205,6 +220,47 @@ export const Footer = memo(
                             </ul>
                         </div>
                     </div>
+                    {partnersLogos && (
+                        <div className="fr-footer__partners">
```
Comment (enguerranws) [inline src/Footer.tsx:224]: Can you use `fr.cx()` to add classes?

File: src/Footer.tsx [hunk 2]
```diff
@@ -205,6 +220,47 @@ export const Footer = memo(
...
+                            <h4 className="fr-footer__partners-title">{t("our partners")}</h4>
+                            <div className="fr-footer__partners-logos">
+                                <div className="fr-footer__partners-main">
+                                    <a
+                                        className="fr-footer__partners-link"
+                                        href={partnersLogos.main.href}
+                                    >
```
Comment (garronej) [inline src/Footer.tsx:231]:
```suggestion
                                    <a
                                        className="fr-footer__partners-link"
                                        href={partnersLogos.main.href}
                                        taget="_blank"
                                    >
```
We probably want to open a new tab don't we?

**Review by garronej — COMMENTED** [review]
Thank you for your PR!  

Please use `fr.cx()` everywhere as mentioned by @enguerranws and we should be able to merge.

**Comment by garronej** [comment]
Thank you for your contribution @gary-van-woerkens !

**Comment by gary-van-woerkens** [comment]
Thank you guys for your reviews and your suggestions !

**Comment by garronej** [comment]
Hum it seems that my suggestion to add target blanck wasn't a good idea after all
Will fix  
image

**Comment by garronej** [comment]
Also @gary-van-woerkens we want to allow users to overload the default classes.  Every classes `fr-xxx__yyyy` should be listed as key in the classes object.  

Checkout this patch:  https://github.com/codegouvfr/react-dsfr/commit/c280023133a85d8198bd24bda40738c62a8ade51  

I'm just letting you know for next time ;)

**Pull request merged**